### ISSUE TITLE
Neural Network Terminology

### ISSUE THREAD

**Issue by chfor183** [chfor183/data_science_articles#48]
## TL;DR

Backpropagation : correction à reculons basé sur le résultat de la sortie (technique Momentum)
Transfer learning : utilise un modèle pré-entrainé pour créer un nouveau modèle sur un ensemble de données réduites
Activation function : fonction qui détermine qu’un perceptron est actif et à quel niveau
Epochs : nombre de fois que le NN s’est entrainé sur l’ensemble des données

Hyperparameters:
   Learning rate : amplitude des changements aux paramètres par itération
   Number of hidden layers : profondeur du réseau de neurones
   Weight initialization : valeur de départ pour la fonction d’activation
   Batch size : quantité d’information traitée avant une mise à jour des paramètres

## Key Takeaways
- 
- 

## Useful Code Snippets
```
function test() {
  console.log("notice the blank line before this function?");
}
```

## Articles/Ressources
